politeiawww/backend.go

```diff
@@ -1567,14 +1567,17 @@ func (b *backend) ProcessNewProposal(np www.NewProposal, user *database.User) (*
 
 		// Add the new proposal to the inventory cache.
 		b.Lock()
-		b.newInventoryRecord(pd.Record{
+		err = b.newInventoryRecord(pd.Record{
 			Status:           pd.RecordStatusNotReviewed,
 			Timestamp:        ts,
 			CensorshipRecord: pdReply.CensorshipRecord,
 			Metadata:         n.Metadata,
 			Files:            n.Files,
 		})
 		b.Unlock()
+		if err != nil {
+			log.Errorf("ProcessNewProposal: could not add record into inventory")
```

Comment:
Same comment as above.

Reply:
yes! Thanks for catching that. I will make the adjustment.